### PR TITLE
Added right margin to arrow and disable/enable highlight animation

### PR DESCRIPTION
**Pull request by berkerbilgi**

### CODE REVIEW

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -540,17 +538,19 @@
...
+				DEVELOPMENT_TEAM = "";
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				INFOPLIST_FILE = ExpandableCell/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.seungyounyi.ExpandableCell;
+				PRODUCT_BUNDLE_IDENTIFIER = com.berkerbilgi.ExpandableCell;
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: This is not fair to the creator of this library to change the package name of the library

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -562,17 +562,19 @@
...
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: This is not fair to the creator of this library to change the package name of the library

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -583,13 +585,15 @@
...
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = 4G7K3CN2JU;
+				DEVELOPMENT_TEAM = "";
 				INFOPLIST_FILE = ExpandableCellDemo/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.seungyounyi.ExpandableCellDemo;
+				PRODUCT_BUNDLE_IDENTIFIER = com.berkerbilgi.ExpandableCellDemo;
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: This is not fair to the creator of this library to change the package name of the library

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -599,13 +603,15 @@
...
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: This is not fair to the creator of this library to change the package name of this library

**Comment by pikaboo** [comment]
@berkerbilgi Please return the meta data of this library to the initial state so that your pull request can be reviewed

**Comment by berkerbilgi** [comment]
I changed meta data. Sorry this is my first contribute.

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -279,14 +279,12 @@
 				TargetAttributes = {
 					19EF94E81F3757AC00085BCA = {
 						CreatedOnToolsVersion = 8.3.3;
-						DevelopmentTeam = 4G7K3CN2JU;
 						LastSwiftMigration = 0830;
-						ProvisioningStyle = Automatic;
+						ProvisioningStyle = Manual;
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -279,14 +279,12 @@
...
 					};
 					19EF94F71F37580B00085BCA = {
 						CreatedOnToolsVersion = 8.3.3;
-						DevelopmentTeam = 4G7K3CN2JU;
-						ProvisioningStyle = Automatic;
+						ProvisioningStyle = Manual;
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -540,8 +538,9 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = 4G7K3CN2JU;
+				DEVELOPMENT_TEAM = "";
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 8]
```diff
@@ -551,6 +550,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.seungyounyi.ExpandableCell;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 9]
```diff
@@ -562,8 +562,9 @@
 			buildSettings = {
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "";
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = 4G7K3CN2JU;
+				DEVELOPMENT_TEAM = "";
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 10]
```diff
@@ -583,13 +585,15 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_STYLE = Manual;
 				DEFINES_MODULE = YES;
-				DEVELOPMENT_TEAM = 4G7K3CN2JU;
+				DEVELOPMENT_TEAM = "";
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 11]
```diff
@@ -583,13 +585,15 @@
...
 				INFOPLIST_FILE = ExpandableCellDemo/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				PRODUCT_BUNDLE_IDENTIFIER = com.seungyounyi.ExpandableCellDemo;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

File: ExpandableCell.xcodeproj/project.pbxproj [hunk 12]
```diff
@@ -599,13 +603,15 @@
...
```
Comment (pikaboo) [inline ExpandableCell.xcodeproj/project.pbxproj]: can you revert these changes as well please ?

**Comment by pikaboo** [comment]
@younatics please make a new pod version
